docs/semgrep-supply-chain/getting-started.md

```diff
@@ -85,6 +85,34 @@ You can configure your CI/CD system to trigger a Semgrep Supply Chain scan whene
  </tr>
 </table>
 
+## Scan a project without lockfiles (beta)
+
+:::info
+Semgrep does not support managed scans of projects without lockfiles.
+:::
+
+Semgrep supports the scanning of Java projects **without lockfiles** if they're built using Maven or Gradle with the help of the Gradle Wrapper. To do so:
+
+1. Ensure that the environment where you are initiating the Semgrep scan has all of the dependencies required to build your project, such as Java and Maven, installed.
+
+2. Initiate a Semgrep scan, ensuring that you include the `--allow-local-builds` flag:
+    ```console
+    semgrep ci --allow-local-builds
+    ```
```

Comment:
```suggestion
    ```
    For existing CI jobs, you may have to edit your configuration file to include this flag.
```